File: 이것이 코딩테스트다/05.이진 탐색/1.순차탐색.js
const texts = [
  5,
  "Dongbin",
  ["Hanul", "Jonggu", "Dongbin", "Taeil", "Sangwook"],
];

const n = texts.shift();
const target = texts.shift();
const array = texts.pop();

const sequentialSearch = (n, target, array) => {
  for (let i = 0; i < n; i++) {
    if (array[i] === target) {
      return i + 1;
    }
  }
};

console.log(sequentialSearch(n, target, array));

File: 이것이 코딩테스트다/05.이진 탐색/1.순차탐색.py
input_data = input().split()
n = int(input_data[0])
target = input_data[1]

array = input().split()

def sequential_search(n, target, array):
  for i in range(n):
    if array[i] == target:
      return i+1
    
print(sequential_search(n, target, array))